cmd/tests/cmd_run_test.go

```diff
@@ -2184,8 +2199,8 @@ func TestBrowserPermissions(t *testing.T) {
 					},
 				},
 			}`,
-			expectedExitCode: 0,
-			expectedError:    "",
+			expectedExitCode: 108,
+			expectedError:    "error building browser on IterStart: launching browser: exec: \"k6-browser-fake-cmd\": executable file not found in $PATH",
```

Comment:
I'm curious if it's the same on windows machines:thinking:

Reply:
Very good spot, i've resolved it in 59055e787215d99637aa9656d55c69df906bbe64